modules/update-conditional-updates.adoc

```diff
@@ -0,0 +1,39 @@
+// Module included in the following assemblies:
+//
+// * updating/updating-cluster-cli.adoc
+// * updating/understanding-upgrade-channels-releases.adoc
+
+[id="update-conditional-upgrade-path{context}"]
+= Updating to a conditional upgrade path
```

Comment:
"to an ... upgrade path" -> "along an ... upgrade path"?  Also "update along a conditional upgrade path" and such below?